instabug/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "37d9bc39-888f-4bec-b8c5-3c137cf88f84",
+  "app_id": "instabug",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and track your mobile app health and performance.",
+    "title": "Instabug",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Use the Instabug widget to monitor the overall performance of your app.",
+        "image_url": "images/instabug-datadog-widget.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Instabug's dashboard gives teams a succinct overview of the app's performance, and potential failures or issues occurring in the app.",
+        "image_url": "images/instabug-app-overview.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "With each point of feedback or bug your users submit, Instabug automatically captures all the details you need to understand why users are facing issues with your code.",
+        "image_url": "images/instabug-bugs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Prioritize resolution by the most impactful crashes to your users.",
+        "image_url": "images/instabug-crashes-list.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Understand how each version of your app release is performing and overall adoption.",
+        "image_url": "images/instabug-releases-page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Aggregate data across device and version to better understand where issues are occurring.",
+        "image_url": "images/instabug-crash-details.png"
+      }
+    ],
+    "classifier_tags": [
+      "Offering::UI Extension",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Alerting",
+      "Category::Monitoring",
+      "Category::Issue Tracking"
+    ]
+  },
+  "assets": {
+    "dashboards": {},
+    "monitors": {},
+    "logs": {},
+    "saved_views": {}
```

Comment:
```suggestion
```
These can be removed since they're empty. Please try re-running the tests. If they fail, please rebase your branch.

1. Go to the github webapp
2. Go to your repositories
3. Select/click on your forked repo of integrations-extras
4. Once you click the forked repository, go to sync fork in the github web--ui
6. Click "update branch"